packages/web/src/components/admin_page/feature_switch_page/render_switch.jsx

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-unused-vars */
+import React, { useState, useEffect } from 'react';
+import { Button } from 'react-bootstrap';
+import Validate from '../../../Global';
+import InvalidClient from '../invalid_client';
+import { useNavigate } from 'react-router-dom';
+import axios from 'axios';
+import { parseToNewAPI, featureSwitchesURL } from '../../../Global';
+import FeatureRow from './row_switch';
+import { OverlayTrigger, Popover, Form, Alert } from 'react-bootstrap';
+import { FsContext } from '../../../feature_switches';
+import { useContext } from 'react';
+
+const popover = (onUpdate, errorMessage) => {
+    return (
+        <Popover id="popover-basic">
+            <Popover.Header as="h3">Want to update or remove?</Popover.Header>
+            <Popover.Body>
+                {errorMessage !== undefined ? <Alert variant="warning">{errorMessage}</Alert> : null}
+                <UpdateSwitch onUpdate={onUpdate} />
+            </Popover.Body>
+        </Popover>
+    );
+};
+const UpdateSwitch = ({ onUpdate }) => {
+    const [newSwitch, setNewSwitch] = useState({
+        is_enabled: 'true',
+    });
+
+    const handleChange = e => {
+        const { name, value } = e.target;
+
+        setNewSwitch({
+            ...newSwitch,
+            [name]: value,
+        });
+    };
+
+    return (
+        <>
+            <Form.Group className="mb-3" controlId="formBasicEmail">
+                <Form.Label>Set a feature:</Form.Label>
+                <Form.Control type="text" placeholder="Paste the url" onChange={handleChange} name="switch_id" />
+                <Form.Text className="text-muted">Add a new feature</Form.Text>
+            </Form.Group>
+            <Form.Group className="mb-3" controlId="formBasicEmail">
+                <Form.Label>Set it to true or false:</Form.Label>
+                <Form.Select onChange={handleChange} name="is_enabled" value={newSwitch.is_enabled}>
+                    <option value="true">True</option>
```

Comment:
If I were you I would have implemented a switch which will show the current state and by clicking on it, it will switch to the !is_enabled state.
If you decide this is out of the scope of this ticket and just unnecessary work leave it as is,